server/src/main/java/io/spine/server/aggregate/AggregateRepository.java

```diff
@@ -616,9 +618,10 @@ protected A play(I id, AggregateHistory history) {
         tx.commitIfActive();
         if (!success) {
             lifecycleOf(id).onCorruptedState(outcome);
-            throw newIllegalStateException("Aggregate %s (ID: %s) cannot be loaded.%n",
-                                           aggregateClass().value().getName(),
-                                           result.idAsString());
+            throw newIllegalStateException(
```

Comment:
Please revert the formatting. The policy is not to split the method and the first param if possible, i.e. if the char count is not too big.